pkg/telemetry/metrics_helper.go

```diff
@@ -27,10 +27,10 @@ func getEndpointSHA(plugin *cli.PluginInfo) string {
 }
 
 // computeEndpointSHAForContext computes the endpoint SHA for based on the target type(context type) used
-func computeEndpointSHAForContext(curCtx map[configtypes.Target]*configtypes.Context, targetType configtypes.Target) string {
+func computeEndpointSHAForContext(curCtx map[configtypes.ContextType]*configtypes.Context, targetType configtypes.Target) string {
 	switch targetType {
 	case configtypes.TargetK8s:
-		ctx, exists := curCtx[configtypes.TargetK8s]
+		ctx, exists := curCtx[configtypes.ContextTypeK8s]
```

Comment:
Should we look for a TAE current context here when there is not K8s current context?

Reply:
Addressed this as part of the latest commit.